tests/unit_tests/fcmp_pp.cpp

```diff
@@ -454,17 +455,17 @@ TEST(fcmp_pp, verify)
                 for (std::size_t i = 0; i < helios_scalar_chunks.size(); ++i)
                     selene_branch_blinds.emplace_back(fcmp_pp::selene_branch_blind());
 
-            if (helios_branch_blinds.empty())
-                for (std::size_t i = 0; i < selene_scalar_chunks.size(); ++i)
-                    helios_branch_blinds.emplace_back(fcmp_pp::helios_branch_blind());
+            std::vector<const uint8_t*> helios_blinds;
+            for (const auto &hbb : helios_branch_blinds)
+                helios_blinds.push_back((uint8_t*)hbb.get());
```

Comment:
This doesn't break the strict aliasing rule since there's an exception for accessing any objects though `char` and `unsigned char` pointers.

Reply:
The latest commit ([4636f99](https://github.com/seraphis-migration/monero/pull/39/commits/4636f997cbf8483ab35bc31459f33f04e861b0da)) removes all usage of uint8_t* for Helios branch blinds. Have a feeling you'll have some thoughts on that approach too. Will stop here before proceeding.

Reply:
Yeah looks good!